starter-code/src/App.js

```diff
@@ -13,6 +15,9 @@ class App extends Component {
         <p className="App-intro">
           To get started, edit <code>src/App.js</code> and save to reload.
         </p>
+        <ul>
```

Comment:
This `<ul>` has no `<li>`. Even if it had some inside of `<FoodList>` the element should probably be inside of the food list component.